pkg/webui/components/breadcrumbs/breadcrumb/breadcrumb.styl

```diff
@@ -15,14 +15,14 @@
 .link
   one-liner()
   text-decoration: none
-  color: $tc-subtle-gray
+  color: $c['grey-700'] !important
```

Comment:
`!important` is usually a code-smell. You should be able to remove it if you define `grey-400` as a default, and then set grey-700 on the last item only.